sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/SamplingResult.java

```diff
@@ -67,6 +67,42 @@ static SamplingResult create(SamplingDecision decision, Attributes attributes) {
         : ImmutableSamplingResult.createSamplingResult(decision, attributes);
   }
 
+  /**
+   * Returns a {@link SamplingResult} corresponding to {@link SamplingDecision#RECORD_AND_SAMPLE}
+   * with no attributes and {@link SamplingResult#getDecision()} returning {@code decision}.
+   *
+   * <p>This is meant for use by custom {@link Sampler} implementations.
+   *
+   * @return A {@link SamplingResult} with empty attributes and the provided {@code decision}.
```

Comment:
```suggestion
   * <p>This is meant for use by custom {@link Sampler} implementations and is equivalent to calling
   * {@code SamplingResult.create(SamplingDecision.RECORD_AND_SAMPLE)}.
   *
   * @return A {@link SamplingResult} with empty attributes and the provided {@code decision}.
```

Let's add something like this to the three methods.

Reply:
I have added them, could you check for me and merge it please. Thanks.